README.md

```diff
@@ -117,6 +117,9 @@ gcloud iam roles create CustomDiskRole \
     --stage="GA"
 ```
 
+#### Azure rule
+The default role `Tag Contributor` can be used to configure the access rights for the pvc-tagger. 
```

Comment:
Can you add a bit about how this only support Azure `disk` csi volumes?

Reply:
I've documented the behaviors hopefully this description will stay up to date in the future.